### PR TITLE
records: Python 2.6 compatibility fix

### PR DESCRIPTION
**Pull request by jalavik**
Signed-off-by: Jan Aage Lavik <jan.age.lavik@cern.ch>

### CODE REVIEW

**Comment by jalavik** [comment]
I don't understand which order kwalitee wants me to put the imports. Anyway, the true fix here is the string formatting.

File: invenio/modules/records/models.py [hunk 1]
```diff
@@ -20,8 +20,8 @@
 """Record models."""
 
 from flask import current_app
-from sqlalchemy.ext.declarative import declared_attr
```
Comment (jirikuncar) [inline invenio/modules/records/models.py]: this was fine

File: invenio/modules/records/models.py [hunk 2]
```diff
@@ -186,9 +186,9 @@ def bibxxx(cls):
 models = []
 
 for idx in range(100):
-    Bibxxx = "Bib{:02d}x".format(idx)
+    Bibxxx = "Bib{0:02d}x".format(idx)
     globals()[Bibxxx] = type(Bibxxx, (db.Model, BibxxxMixin), {})
-    BibrecBibxxx = "BibrecBib{:02d}x".format(idx)
+    BibrecBibxxx = "BibrecBib{0:02d}x".format(idx)
```
Comment (jirikuncar) [inline invenio/modules/records/models.py]: thanks

**Comment by jirikuncar** [comment]
`Reviewed-by: me`

**Comment by jalavik** [comment]
Thanks. Updated.

**Pull request merged**